config/samples/policies_v1alpha3_policyserver.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: policies.kubewarden.io/v1alpha3
+kind: PolicyServer
+metadata:
+  name: policyserver
+spec:
+  image: ghcr.io/kubewarden/policy-server:v1.0.0
+  replicas: 1
+  env:
+    - name: KUBEWARDEN_LOG_LEVEL
+      value: debug
+    - name: KUBEWARDEN_LOG_FMT
+      value: jaeger
+  annotations:
+    sidecar.jaegertracing.io/inject: default
+  serviceAccountName: sa
```

Comment:
Since this is a sample resource, should we create the `sa` service account here too? So it refers to something that exist.